src/main/java/model/LocustRequestData.java

```diff
@@ -0,0 +1,122 @@
+package model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class LocustRequestData {
```

Comment:
For what is this class?

Reply:
The LocustRequestData is the model object that manage the request data from the CSV. Once the LocustReportReader read the csv and create the LocustRequestData, we have an object with all the information results of the request with this class